.github/workflows/release.yml

```diff
@@ -0,0 +1,108 @@
+on:
+  push:
+    tags:
+    - 'v*'
+
+name: Upload Release Asset
+
+jobs:
+  build:
+    name: Upload Release Asset
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Build project
+        run: |
+          bash ./scripts/artifacts.sh
+      - name: Create Release
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: ${{ github.ref }}
+          release_name: Release ${{ github.ref }}
+          draft: false
+          prerelease: false
+      - name: Upload Cryptography Core Source
```

Comment:
Github automatically attaches source code with releases. Is there any reason why we're doing this manually?

Reply:
I believe the intention is to have source code packages for the specific components.  Still, given that there is a parent source code artifact already (as you pointed out) I agree that this seems to provide little value.

Reply:
Yeah, I wanted to publish the source of each project separately since they can be used independently of each other (as if they were in separate repositories).